### PR TITLE
kotsadm 1.2.0

### PR DESCRIPTION
**Pull request by laverya**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -154,9 +154,9 @@ Enable the aws, nodeless and calico plugins, and disable prometheus, weave, rook
     +#CONTOUR_VERSION=0.14.0
      REGISTRY_VERSION=2.7.1
     -PROMETHEUS_VERSION=0.33.0
-    -KOTSADM_VERSION=1.1.0
+    -KOTSADM_VERSION=1.2.0
     +#PROMETHEUS_VERSION=0.33.0
-    +#KOTSADM_VERSION=1.1.0
+    +#KOTSADM_VERSION=1.2.0
```
Comment (emosbaugh) [inline README.md]: why is this here? i know you didnt add it

Comment (emosbaugh) [reply README.md]: ah, opening up the file i have more context

Comment (laverya) [reply README.md]: I have no idea!
But this didn't seem like the time to fix that, and IMO it should still be kept up to date
